Add new resource `Datadog::Monitors::DowntimeSchedule` (#271)
DataDog/datadog-cloudformation-resources: datadog-monitors-downtimeschedule-handler/docs/monitoridentifier.md

# Datadog::Monitors::DowntimeSchedule MonitorIdentifier

## Syntax

> To declare this entity in your AWS CloudFormation template, use the following syntax:

Reviewer: ```suggestion
To declare a `Datadog::Monitors::DowntimeSchedule MonitorIdentifier` in your AWS CloudFormation template, use the following syntax:
```

Author: This is autogenerated as well
